lib/bin/reassign_m8.py

```diff
@@ -0,0 +1,52 @@
+import argparse
+
+import pandas as pd
+
+
+def main(
+    m8_filepath: str,
+    reads_to_contigs_filepath: str,
+    output_filepath: str,
+):
+    m8 = pd.read_csv(m8_filepath, sep="\t", names=[
+        "read_or_contig_id",
+        "accession",
+        "pid",
+        "alignment_length",
+        "mismatches",
+        "gap_openings",
+        "q_start",
+        "q_end",
+        "s_start",
+        "s_end",
+        "evalue",
+        "bit_score",
+    ])
+
+    reads_to_contigs = pd.read_csv(reads_to_contigs_filepath, sep="\t", names=[
```

Comment:
I think there can be duplicates in the reads_to_contigs tsv due to a read being able to align multiple times to a contig.

Reply:
Got rid of this file